crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -632,24 +632,36 @@ impl CumulocityConverter {
         &mut self,
         message: &MqttMessage,
     ) -> Result<Vec<MqttMessage>, ConversionError> {
-        let operation = C8yOperation::from_json(message.payload.as_str()?)?;
-        let device_xid = operation.external_source.external_id;
-        let cmd_id = self.command_id.new_id_with_str(&operation.op_id);
+        // Smartrest messages on c8y/devicecontrol/notifications can contain multiple operations in a single MQTT
+        // message, so split them
```

Comment:
Terminoligy correction. The messages arrving on `c8y/devicecontrol/notifications` is **JSON over MQTT** messages. **SmartREST** is the comma separated csv-like messages.

```suggestion
        // JSON over MQTT messages on c8y/devicecontrol/notifications can contain multiple operations in a single MQTT
        // message, so split them
```